src/ui/layouts/stack-detail-layout.tsx

```diff
@@ -23,9 +23,19 @@ import {
   IconInfo,
   TabItem,
   Tooltip,
+  getStackImg,
 } from "../shared";
 import { AppSidebarLayout } from "./app-sidebar-layout";
 
+const getStackTypeTitle = (stack: DeployStack) => {
+  const stackType = getStackType(stack);
+  if (stackType === "self_hosted") {
+    return `${capitalize(stackType)} (${stack.awsAccountId})`;
```

Comment:
`capitalize` won't cut it. `self_hosted` will become `Self_hosted`. I'd just hardcode it since we're specifically checking the string.

```suggestion
    return `Self-Hosted (${stack.awsAccountId})`;
```